feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -577,6 +579,26 @@ func TestControllerCardEmpty(t *testing.T) {
 	}
 }
 
+// If the component has a leafref for a subcomponent, that leaf must exist in the list of
+// components.
+func verifySubcomponentNameExists(c *oc.Component, components []*oc.Component, t *testing.T) {
```

Comment:
Optional:
* Best practice is to also start unexported function comments with the same name. If it ever gets exported (say moved into a library), then it would match the style guide.
* Also recommend to name this `validateSubcomponentsExistAsComponents` to reflect the action more clearly.

Reply:
Sounds good, thanks for the suggestions! I have updated the name and included it in the comment.